Der Ast aller Äste... Die Fight-Klasse

SariusRu opened the pull request:
Die Fight-Klasse mit MEINER Dice-Klasse!
Ein <3 für Felix
by Aaron und Samuel

Von Felix: fixes #32
Dieses Keyword verknüpft diesen Pull Request mit dem Issue Nr. 32.

SariusRu committed:
fight

erklärt sich von selbst

SariusRu committed:
Delete Fight.java

SariusRu committed:
Die neue Fight-Klasse (Ein <3 für Felix)

Zocker1999NET committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Ast

Update des Branches erfolgt, um das Debugging zu erleichtern.

Zocker1999NET commented:
Der Ast Branch wurde nun zuerst vom Master aktualisiert, um das Debugging zu ermöglichen. Es gingen dabei keine Änderungen von eurer Seite verloren.
Nach dem Start des eurer Welt und dem Start der Simulation (über den Run Button unten) tritt bei Greenfoot folgender Fehler auf:

```
java.lang.NullPointerException at Fight.act(Fight.java:29)
```

**Als Tipp**: Der Fehler tritt auf, da die Provinz-IDs erst bei 1 anfangen, dies steht auch so in der Architektur.
Ich schlage euch vor, ab sofort mit dieser Version des Branches weiterzuarbeiten. Bitte testet die Version danach _selbstständig weiter_, bis alles so funktioniert, wie ihr es haben wollt. Falls ihr Fragen habt zum weiteren Testen/Debuggen könnt ihr die mir stellen über _Mail/Skype/WhatsApp_.
Sobald ihr den Fehler und eventuell doch auch weitere behoben habt, bitte einen Kommentar bei diesem Pull Request abgeben oder sich bei mir über WhatsApp melden. (Der Pull Request muss nicht geschlossen und wieder geöffnet werden, da dieser sich automatisch aktualisiert.)

SariusRu committed:
Ein <3 Für Felix

Zocker1999NET committed:
Map_World.java Erneuerung Teil 1

Der Pull Request zeigt an, dass diese Datei einen Unterschied zur der Version hat, die im Master Branch liegt.
Da eigentlich kein Unterschied vorhanden ist und ich dies bereits zuvor gemerkt habe, vermute ich das Problem beim Format des Zeilenendes (Wiki: CRLF und LF). Deshalb wird nun diese Datei gleich erneut hochgeladen, um diesen vermuteten Fehler zu beheben/auszuschließen.

Zocker1999NET committed:
Map_World.java Erneuerung Teil 2

Siehe Commit: f2e2615779eab8c60e1da6cfd3e7c17aee0d409e
